server/tests/server/data/parser-auror_testssl-cert.pem

```diff
@@ -0,0 +1,74 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
not acceptable to use real certificate here

Reply:
Why? We may use some other cert, but why is real cert an issue?

Reply:
All testing data should be either fake or clean and not referencing any real-world application as much as possible to prevent prevent clash or misunderstanding in the future. There may come some code which uses this testing certificate and make outbound connections somewhere based on the data.

As long as we don't have some serious obstacles to use this specific certificate or we cannot forge our own testing cert please use dummy self-signed cert if possible.